src/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const EXPECTED_PERMISSIONS_BLOCK = [
+  "  permissions:",
+  "    actions: write",
+  "    checks: write",
+  "    contents: read",
+  "    discussions: write",
+  "    pull-requests: write",
+  "    statuses: write",
+  "    deployments: read",
+  "",
```

Comment:
Copied this from the docs page + added the deployments: read permission just in case.